Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -0,0 +1,57 @@
+import KsApi
+import Library
+import UIKit
+
+internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControllerDataSource {
+  private let viewControllers: [UIViewController]
+
+  internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
+    let backedController = BackerDashboardProjectsViewController()
+    let savedController = BackerDashboardProjectsViewController()
+
+    if let delegate = delegate as? BackerDashboardProjectsViewControllerDelegate {
+      backedController.configureWith(delegate: delegate, projectsType: .backed, sort: sort)
+      savedController.configureWith(delegate: delegate, projectsType: .saved, sort: sort)
+    }
+
+    var controllers: [UIViewController] = []
+    controllers[BackerDashboardTab.backed.rawValue] = backedController
+    controllers[BackerDashboardTab.saved.rawValue] = savedController
+    self.viewControllers = controllers
+  }
+
+  internal func controllerFor(tab: BackerDashboardTab) -> UIViewController? {
+    return self.viewControllers[tab.rawValue]
```

Comment:
and then to do this we would need to look up the index of the tab from `allTabs`